tests/bdd/features/lti_certification/section_1_invalid_launch_requests.feature

```diff
@@ -2,10 +2,41 @@ Feature: Section 1 - Invalid Launch Requests
 
   This section comprises tests using invalid launch requests.
 
+  Background:
+    Given fixtures are located in '/lti_certification_1_1/section_1'
+      And the OAuth 1 consumer key is 'Hypothesis4dd96539c449ca5c3d57cc3d778d6bf3'
+      And the OAuth 1 shared secret is 'TEST_SECRET'
+      And I create an LMS DB 'ApplicationInstance'
+        | Field            | Value                                      |
+        | consumer_key     | Hypothesis4dd96539c449ca5c3d57cc3d778d6bf3 |
+        | shared_secret    | TEST_SECRET                                |
+        | lms_url          | test_lms_url                               |
+        | requesters_email | test_requesters_email                      |
+      And I load the fixture 'most_common.ini' as 'params'
+
   @v1.0 @v1.1 @v1.2 @required
   Scenario: Test 1.1 - No resource_link_id provided
     # Expected result: Return user to the Tool Consumer with an error message
 
+    Given I update the fixture 'params' with
+	  | Key                     | Value               |
+	  | custom_link_setting_url | $LtiLink.custom.url |
```

Comment:
Does the formatting of `$LtiLink.custom.url` mean something or it is it a literal value?

Reply:
It's a string. I've no idea why it's here. I think the answer is "because whoever wrote the tests didn't design the fixtures very well". I wanted out fixtures to 100% match the LTI ones.

The only magic values are "*MISSING*" and "*NONE*".